code-challenges.test.js

```diff
@@ -25,24 +25,94 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+// 6. Expect to see object3 output "-7 is not divisible by 3"
+
+// JEST TEST
+    // describe divBy3 = (number) =>{
+    //     it(returns object divisible by 3 or not divisible by 3, () => {
+    //         expect if divBy is run "object1 is divisible by 3"
+    //         expect if divBy3 is run "object2 is divisible by 3"
+    //         expect if divBy3 is run "object3 is not divisible by 0"
+    //     }
+    // }
+// JEST TEST 
+
 const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// // Expected output: "-7 is not divisible by three"
+
+
+describe("divBy3", () => {
+    it("returns whether a given number is divisible by three or not", () => {
+        expect (divBy3(object1)).toEqual("15 is divisible by three")
+        expect (divBy3(object2)).toEqual("0 is divisible by three")
+        expect (divBy3(object3)).toEqual("-7 is not divisible by three")
+    
+    }) 
+})
+
+const divBy3 = (x) =>  {
+    if (x.number % 3 === 0) {
+        return `${x.number} is divisible by three`
+    } else {
+        return `${x.number} is not divisible by three`
+    }
+}
+
+
+
+
+
 
 // b) Create the function that makes the test pass.
 
+
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Pseudo-code
+// 1. Create a test that will capitalize the first letter of each word in the given arrays
+// 2. Describe the function shiftToCap
+// 3. shiftToCap takes in the strings in the arrays
+// 4. expect shiftToCap to capitalize the first letter in each array.
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+// Jest Test
+describe("shiftToCap", () => {
+    it("returns an array with the first letter of each element capitalized", () => {
+        expect (shiftToCap(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect (shiftToCap(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
+
+// Upon creating the test shown above, I received a good Failure, because the function shiftToCap has not yet been defined. Specific error is:
+//  SyntaxError: Unexpected identifier 'shiftToCap'
+// Since the function has not yet been defined, it cannot perform the function call.
+
+// const shiftToCap = [array] => {
+//     for (i = 0; i < array.length; i++) {
```

Comment:
This one is so close, you just need to upperCase the first index and then either slice or add a substring method to return the rest of the string as lowercase with the first letter capitalized. Give it a try!